target/tools/files/README.TOOLS

```diff
@@ -0,0 +1,10 @@
+This is part of the OpenWrt SDK. It contains just the build tools created
+by the buildroot.
+
+It can be copied to another openwrt buildroot and unpacked there,
+the paths for default staging_dir and build_dir are preserved.
+
+This should be unpacked directly in the top level directory (openwrt folder)
+example:
+
+	$ xzcat openwrt-tools.Linux-x86_64.tar.xz | tar xf - -C openwrt
```

Comment:
Did you test that this actually recycles stamps? Meaning that you unpack it at varying path and it doesn't rebuild everything from tools/?

Reply:
that is the plan, it needs more testing though

Reply:
I tested with one package, xz so far, which does not rebuild

like this

```
make dirclean
make tools/xz/compile
make target/tools/compile NO_DEPS=1
cp bin/targets/ath79/generic/openwrt-tools.Linux-armv7l.tar.xz .
make dirclean
xzcat openwrt-tools.Linux-armv7l.tar.xz | tar xf -
make tools/xz/compile
```

Reply:
Wonder if I should test this on my github action 

Can you check if `make tools/install V=s` cause tools recompilation?

Reply:
@aparcar btw another option if we decide to host such tar in openwrt downloads would be use github actions cache 

But IMHO the idea of adding this to openwrt downloads will be NACK as we already have the sdk

Reply:
Mhhh don't know testing this and tools are recompiled.

```
find build_dir/host | xargs touch
sleep 1
find staging_dir/host/stamp | xargs touch
```

using this workaround tools are not recompiled.